dashboard/src/components/Catalog/Catalog.tsx

```diff
@@ -143,7 +143,7 @@ class Catalog extends React.Component<ICatalogProps, ICatalogState> {
                 </div>
               </div>
             )}
-            <div className={this.shouldRenderOperators() ? "col-10" : ""}>
+            <div className={this.shouldRenderOperators() ? "col-10" : "col-12"}>
```

Comment:
Why is this dependent on whether we're rendering operators? That is, why not use the same width ("col-10") either way? If it's because the filter is not present, then we should use a function that makes that obvious, perhaps?

Reply:
yes, it's because if there are no operators we are skipping the column that allows you to filter by chart/operator. This function is used several times (line 103 and 122), I guess it's more obvious in the full context of the function.